### PR TITLE
blog: meet our contributors APAC China region

### PR DESCRIPTION
**Pull request by jayesh-srivastava**
This PR adds the third blog post for China region, in the series of ["Meet Our Contributors - APAC"](https://docs.google.com/document/d/1zB-FOvfbnKbiWlGgInnDB06hDfLsTQuuKsXQCqc-lL0/edit#heading=h.yr3qa2ko7xq) blog series.

Reference issue: https://github.com/kubernetes/community/issues/4469#issuecomment-942422348

### CODE REVIEW

File: content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+ layout: blog
+ title: "Meet Our Contributors - APAC (China region)"
+ date: 2022-07-11T12:00:00+0000
+ slug: meet-our-contributors-chn-ep-03
+ canonicalUrl: https://www.kubernetes.dev/blog/2022/07/11/meet-our-contributors-chn-ep-03/
+ ---
```
Comment (sftim) [inline content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md:7]:
```suggestion
---
layout: blog
title: "Meet Our Contributors - APAC (China region)"
date: 2022-07-11T12:00:00+0000
slug: meet-our-contributors-chn-ep-03
canonicalUrl: https://www.kubernetes.dev/blog/2022/07/11/meet-our-contributors-chn-ep-03/
---
```

File: content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+**Authors & Interviewers:** [Avinesh Tripathi](https://github.com/AvineshTripathi), [Debabrata Panigrahi](https://github.com/Debanitrkl), [Jayesh Srivastava](https://github.com/jayesh-srivastava), [Priyanka Saggu](https://github.com/Priyankasaggu11929/), [Purneswar Prasad](https://github.com/PurneswarPrasad), [Vedant Kakde](https://github.com/vedant-kakde)
+
+---
+
+Hello, everyone 👋
+
+Welcome back to the third episode of the "Meet Our Contributors" blog post series for APAC.
+
+This post features four outstanding contributors from  China, who have played diverse leadership and community roles in the upstream Kubernetes project.
```
Comment (windsonsea) [inline content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md:17]:
```suggestion
This post features four outstanding contributors from China, who have played diverse leadership and community roles in the upstream Kubernetes project.
```
two spaces there 😺

File: content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+So, without further ado, let's get straight to the article.
+
+
+## [Andy Zhang](https://github.com/andyzhangx)
+
+Andy Zhang currently works for Microsoft China at the Shanghai site. His main focus is on Kubernetes storage drivers. Andy started contributing to Kubernetes about 5 years ago.
+
+He states that as he is working in Azure Kubernetes Service team and spends most of his time contributing to the Kubernetes community project. Now he is the main contributor of quite a lot Kubernetes subprojects such as Kubernetes cloud provider code.
```
Comment (windsonsea) [inline content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md:26]: A little confused with `Kubernetes cloud provider code` if I localize this clause. Doe it refer to contributing code for Kubernetes cloud providers?

File: content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+Currently Jintao is a reviewer for the [ingress-nginx](https://kubernetes.github.io/ingress-nginx/) project.
+
+He suggests keeping track of job opportunities at open source companies so that you can find one that allows you to contribute full time. For new contributors Jintao says that if anyone wants to make a significant contribution to an open source project, then they should choose the project based on their interests and should generously invest time.
+
+
+---
+
+
+If you have any recommendations/suggestions for who we should interview next, please let us know in the #sig-contribex channel on the Kubernetes Slack. Your suggestions would be much appreciated. We're thrilled to have additional folks assisting us in reaching out to even more wonderful individuals of the community.
```
Comment (windsonsea) [inline content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md:67]:
```suggestion
If you have any recommendations/suggestions for who we should interview next, please let us know in the [#sig-contribex channel](https://kubernetes.slack.com/archives/C1TU9EB9S) on the Kubernetes Slack. Your suggestions would be much appreciated. We're thrilled to have additional folks assisting us in reaching out to even more wonderful individuals of the community.
```

**Review by windsonsea — COMMENTED** [review]
pick nits 🔢

File: content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md (hunk 3 above)
Comment (Debanitrkl) [reply content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md:26]: Yeah cloud provider coder here means this repo https://github.com/kubernetes-sigs/cloud-provider-azure

File: content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
+---
+layout: blog
+title: "Meet Our Contributors - APAC (China region)"
+date: 2022-07-11T12:00:00+0000
+slug: meet-our-contributors-chn-ep-03
```
Comment (Sea-n) [inline content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md:5]: It was `meet-our-contributors-india-ep-01` for first article.

```suggestion
slug: meet-our-contributors-china-ep-03
```

File: content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+Hello, everyone 👋
+
+Welcome back to the third episode of the "Meet Our Contributors" blog post series for APAC.
+
+This post features four outstanding contributors from  China, who have played diverse leadership and community roles in the upstream Kubernetes project.
+
+So, without further ado, let's get straight to the article.
+
+
```
Comment (Sea-n) [inline content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md:21]: I think it's unnecessary for two newlines.

File: content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+Paco Xu works at DaoCloud, a Shanghai-based cloud-native firm. He works with the infra and the open source team, focusing on enterprise cloud native platforms based on Kubernetes.
+
+He started with Kubernetes in early 2017 and his first contribution was in March 2018. He started with a bug that he found, but his solution was not that graceful, hence wasn't accepted. He then started with some good first issues, which helped him to a great extent. In addition to this, from 2016 to 2017, he made some minor contributions to Docker.
+
+Currently, Paco is a reviewer for `kubeadm` (a SIG Cluster Lifecycle product), and for SIG Node.
+
+Paco says that you should contribute to open source projects you use. For him, an open source project is like a book to learn, getting inspired through discussions with the project maintainers.
+
+> In my opinion, the best way for me is learning how owners work on the project.
+## [Jintao Zhang](https://github.com/tao12345666333)
```
Comment (Sea-n) [inline content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md:53]: Is it related? Maybe add a newline between would be better.

File: content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (jayesh-srivastava) [reply content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md:26]: @divya-mohan0209 I guess `He states that he works in the Azure Kubernetes Service team and spends most of his time contributing to the Kubernetes community project.` will do.

File: content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+
+## [Andy Zhang](https://github.com/andyzhangx)
+
+Andy Zhang currently works for Microsoft China at the Shanghai site. His main focus is on Kubernetes storage drivers. Andy started contributing to Kubernetes about 5 years ago.
+
+He states that as he is working in Azure Kubernetes Service team and spends most of his time contributing to the Kubernetes community project. Now he is the main contributor of quite a lot Kubernetes subprojects such as Kubernetes cloud provider code.
+
+His open source contributions are mainly self-motivated. In the last two years he has mentored a few students contributing to Kubernetes through the LFX Mentorship program, some of whom got jobs due to their expertise and contributions on Kubernetes projects.
+
+Andy is an active member of the China Kubernetes community. He adds that the Kubernetes community has a good guide about how to become members, code reviewers, approvers and finally when he found out that some open source projects are in the very early stage, he actively contributed to those projects and became the project maintainer.
```
[the remark this answers is not in the document]
Comment (jayesh-srivastava) [reply content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md:30]: @divya-mohan0209 Here we are talking about the Kubernetes subprojects. I see Andy is the maintainer for some Kubernetes-SIGs and Kubernetes-CSI projects.

File: content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md [hunk 9]
```diff
@@ -0,0 +1,70 @@
+---
+layout: blog
+title: "Meet Our Contributors - APAC (China region)"
+date: 2022-07-11T12:00:00+0000
```
[the remark this answers is not in the document]
Comment (jayesh-srivastava) [reply content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md:4]: @sftim @Debanitrkl Are we changing the format here?

File: content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md (hunk 3 above)
Comment (jayesh-srivastava) [reply content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md:26]: @windsonsea Any suggestions for this one or will the initial sentence work?

Comment (jayesh-srivastava) [reply content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md:26]: @windsonsea Any suggestions for this one or will the initial sentence work?

[the remark this answers is not in the document]
Comment (jayesh-srivastava) [reply content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md:26]: @divya-mohan0209 Any suggestions here?

File: content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (jayesh-srivastava) [reply content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md:30]: @divya-mohan0209 Any suggestions here?

**Comment by sftim** [comment]
Repeating https://github.com/kubernetes/website/pull/34865#issuecomment-1177659748

> This PR is a mirroring request - the upstream article is the one from [kubernetes/contributor-site#314](https://github.com/kubernetes/contributor-site/pull/314)
> 
> To be extra helpful, make suggestions to [kubernetes/contributor-site#314](https://github.com/kubernetes/contributor-site/pull/314) - and only if the change is material. **Try to avoid suggesting only trivial edits.**

File: content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md [hunk 10]
```diff
@@ -0,0 +1,70 @@
+---
+layout: blog
+title: "Meet Our Contributors - APAC (China region)"
+date: 2022-08-15
+slug: meet-our-contributors-china-ep-03
+canonicalUrl: https://www.kubernetes.dev/blog/2022/07/11/meet-our-contributors-chn-ep-03/
```
Comment (sftim) [inline content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md:6]:
```suggestion
canonicalUrl: https://www.kubernetes.dev/blog/2022/08/15/meet-our-contributors-chn-ep-03/
```

File: content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md (hunk 3 above)
Comment (sftim) [reply content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md:25]: We'll use the wording from the upstream article (review feedback should go there).

[the remark this answers is not in the document]
Comment (sftim) [reply content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md:25]: We'll use the wording from the upstream article (review feedback should go there).

File: content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (sftim) [reply content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md:29]: We'll use the wording from the upstream article (review feedback should go there).

**Comment by sftim** [comment]
/lgtm
/approve

/hold
Waiting for original article to publish

**Comment by sftim** [comment]
Path fixup: #35978

**Pull request merged**